scripts/stablex/utilities.js

```diff
@@ -2,6 +2,33 @@ const BN = require("bn.js")
 const { waitForNSeconds } = require("../../test/utilities.js")
 const token_list_url = "https://raw.githubusercontent.com/gnosis/dex-js/master/src/tokenList.json"
 
+const fetchTokenInfo = async function(exchangeContract, tokenIds, artifacts) {
+  const ERC20 = artifacts.require("ERC20Detailed")
+  console.log("Fetching token data from EVM")
+  const tokenObjects = {}
+  for (const id of tokenIds) {
+    const tokenAddress = await exchangeContract.tokenIdToAddressMap(id)
+    const tokenInstance = await ERC20.at(tokenAddress)
+    let tokenInfo
+    try {
+      tokenInfo = {
+        id: id,
+        symbol: await tokenInstance.symbol.call(),
+        decimals: (await tokenInstance.decimals.call()).toNumber(),
+      }
+    } catch (err) {
+      tokenInfo = {
+        id: id,
+        symbol: "UNKNOWN",
+        decimals: "18",
```

Comment:
This seems quite hacky and somewhat incorrect. Shouldn't the caller of this function decide how to handle tokenIds that don't have any info?